airbyte-integrations/bases/debezium-v1-9-6/src/main/java/io/airbyte/integrations/debezium/internals/PostgresConverter.java

```diff
@@ -69,13 +72,28 @@ public void converterFor(final RelationalColumn field, final ConverterRegistrati
       registerMoney(field, registration);
     } else if (BYTEA_TYPE.equalsIgnoreCase(field.typeName())) {
       registerBytea(field, registration);
+    } else if (JSONB_TYPE.equalsIgnoreCase(field.typeName())) {
+      registerJsonb(field, registration);
     } else if (Arrays.stream(NUMERIC_TYPES).anyMatch(s -> s.equalsIgnoreCase(field.typeName()))) {
       registerNumber(field, registration);
     } else if (Arrays.stream(ARRAY_TYPES).anyMatch(s -> s.equalsIgnoreCase(field.typeName()))) {
       registerArray(field, registration);
     }
   }
 
+  private void registerJsonb(RelationalColumn field, ConverterRegistration<SchemaBuilder> registration) {
+    registration.register(SchemaBuilder.string().optional(), x -> {
+      if (x == null) {
+        return DebeziumConverterUtils.convertDefaultValue(field);
+      }
+      try {
+        return new ObjectMapper().readTree(x.toString()).toString();
```

Comment:
@edgao replaced ObjectMapper to the global variable
I use this construction here not for json validation, but for the result to be similar for СDC and standard sync
The call here is just toString() adds unnecessary spaces to json, ObjectMapper doesn't.
For example
```
                      x.toString()                                           ===> [1, 2, 3]
ObjectMapper().readTree(x.toString()).toString()  ===> [1,2,3]
```

Reply:
wait, what are we putting into the raw table? Is it something like this:
```json
{
  "the_jsonb_field": [1,2,3]
}
```
or more like this:
```json
{
  "the_jsonb_field": "[1,2,3]"
}
```

Reply:
> wait, what are we putting into the raw table? Is it something like this:
> 
> ```json
> {
>   "the_jsonb_field": [1,2,3]
> }
> ```
> 
> or more like this:
> 
> ```json
> {
>   "the_jsonb_field": "[1,2,3]"
> }
> ```

@edgao for STANDARD sync it's 
```
{
  "the_jsonb_field": [1,2,3]
}
```
for CDC 
```
{
  "the_jsonb_field": "[1,2,3]"
}
```

Reply:
As a user, this is both unexpected and problematic - it means we can’t effectively use CDC with large jsonb columns and the Redshift destination, as strings have a much shorter maximum supported size than objects. I was hoping to switch one of our large jsonb tables from full refresh to CDC soon, but sounds like that won’t actually work. (The build of the source connector based on this branch is otherwise running great!)

Reply:
@VitaliiMaltsev what's the current CDC behavior? I think it's ok to move forward and make another issue if it was strings and it's still strings. That is, this only improves the beahvior. Of course, if we could make it the same all the better.

Reply:
@bleonard CDC behaviour is still strings